Rakefile

```diff
@@ -143,7 +143,7 @@ namespace :test do
     Dir.mkdir("carthage_test")
     File.write(File.join("carthage_test", "Cartfile"), "git \"file://#{Dir.pwd}\" \"HEAD\"")
     Dir.chdir "carthage_test" do
-      sh "carthage bootstrap --platform 'iOS'"
+      sh "../scripts/carthage.sh bootstrap --platform 'iOS'"
```

Comment:
👀 was this failing before?

Reply:
In the latest development environment, we're having problems with the xcframework, so we need to use the carthage.sh script.

There are a lot of issues related to Carthage, so I think it would be a good idea to use SPM to set up a project.